### DISCUSSION TITLE
run_simulation error

### DISCUSSION THREAD

**Discussion by mikatakahashi** [IBAHCM/RPiR#249] [Archive]
Hi @soniamitchell 
I get an error message saying:

Error in run_simulation(step_function = timestep_deterministic_SIR, initial.pop = population.df,  : 
  object 'ended' not found

when I try to run simulation of deterministic SIR model. Should I avoid using run_simulation when I try deterministic model?

**Comment by soniamitchell** [comment]
Hey @mikatakahashi, have you added `timestep_deterministic_SIR()` to your package?

**Reply by mikatakahashi** [reply]
yes. I created new function file in R and devtooled it.

**Reply by soniamitchell** [reply]
Could you push your stuff to GitHub so I can see? 👍

**Reply by mikatakahashi** [reply]
Yes! It should be in MikaTakahashiSeries03 with d0306_mean__SIR.R in demo file.

**Reply by soniamitchell** [reply]
Got to go out, will be back in a bit. The error is coming from line 53 of your script though. `run_simulation()` is looking for `ended` but can't find it, I think.
